tests/components/xiaomi_miio/test_vacuum.py

```diff
@@ -521,17 +522,23 @@ async def setup_component(hass, entity_name):
     """Set up vacuum component."""
     entity_id = f"{DOMAIN}.{entity_name}"
 
-    await async_setup_component(
-        hass,
-        DOMAIN,
+    hass.config.components.add(XIAOMI_DOMAIN)
```

Comment:
Let's set up the integration with `hass.config_entries.async_setup` instead.

Reply:
Could you provide me with an example?
I can't find it...

Reply:
```py
await hass.config_entries.async_setup(entry.entry_id)
await hass.async_block_till_done()
```

Reply:
Sorry for not getting it.
but the entry has not yet been made right?
So I don't have an entry_id yet and if I supply a new one the async_setup will fail I think...

I search the HomeAssistant/Core github but I don't get any search results for `await hass.config_entries.async_setup`

Reply:
I get 355 results. :smile:

Reply:
Got it, searching locally worked, for some reason searching online on the github website fails....

Reply:
Done